### DISCUSSION TITLE
cpp_lib.json not found after new build of xformers

### DISCUSSION THREAD

**Discussion by aifartist** [facebookresearch/xformers#642] [Q&A] [answered]
@danthe3rd 
You did a change that may be the cause of a failure I see.
Installing torch 1.13.1 and related packages in a venv and then building xformers results in:
      File "/home/dwood/automate/a1tst/venv/lib/python3.10/site-packages/xformers/_cpp_lib.py", line 114, in _register_extensions
    with open(cpp_lib_json, "r") as fp:
FileNotFoundError: [Errno 2] No such file or directory: '~/automate/a1tst/venv/lib/python3.10/site-packages/xformers/cpp_lib.json'

There happens to be a cpp_lib.json file in ~/automate/a1tst/venv/xformers but not in the python site-packages directories.  My automation creates everything clean but now I'm getting this problem.

Any ideas?  Manually copying the file from where I found it to where it was looking seems to work.

**Comment by danthe3rd** [comment, marked as answer]
There is an issue about that already: https://github.com/facebookresearch/xformers/issues/641
